.gitattributes

```diff
@@ -0,0 +1,19 @@
+## git
+*.git export-ignore
+*.gitignore export-ignore
+*.gitattributes export-ignore
+
+## linter setup
+.prospector.yml export-ignore
+.travis.yml export-ignore
+setup.cfg export-ignore
+
+## docs sources
+doc_theme/ export-ignore
+docs/ export-ignore
+mkdocs.yml export-ignore
+
+## unit testing
+tests/ export-ignore
+run_tests.sh export-ignore
+
```

Comment:
I am unaware of how Package Control does its packaging.  So this actually excludes these from the package?

Also, including documents isn't always a bad thing as people can look at it locally if they like.

Reply:
Package Control runs an `git archive` command via GitHub API. All files in the `.gitattributes` file which are marked with `export-ignore` are therefore not part of a sublime-package.

Leaving docs untouched might not be a bad idea. As they are part of a zip I wonder who finds them there?

Reply:
I don't know, but I generally view shipping them with the product makes sense. Whether anyone finds or looks at them or not isn't a concern for me.

Reply:
Are we sure this stuff doesn't break Travis? I'm getting an error about a missing `.travis.yml` file: https://travis-ci.org/facelessuser/FuzzyFileNav/builds/296883556?utm_source=github_status&utm_medium=notification. I'm trying to restart the build to see what happens.

Reply:
I suspect maybe the way this pull was updated broke Travis's ability to run.  I may just have to do this on a separate branch to verify changes.